### ISSUE TITLE
Problem with help for @everywhere

### ISSUE THREAD

**Issue by andreasnoack** [JuliaLang/julia#11690]
On latest master
```jl
help?> @everywhere
Symbol not found. Falling back on apropos search ...
ERROR: MethodError: `convert` has no method matching convert(::Type{Regex}, ::ASCIIString, ::UInt32)
This may have arisen from a call to the constructor Regex(...),
since type constructors fall back to convert methods.
Closest candidates are:
  Regex(::AbstractString, ::Integer, ::Integer)
  Regex(::AbstractString, ::AbstractString)
  Regex(::AbstractString)
  ...
 in apropos at help.jl:150
 in help at help.jl:133
 in help at help.jl:91
 in help at help.jl:184
```

**Comment by tkelman** [comment]
see #11633, apropos needs to be updated for the changed `Regex` constructor and it apparently needs more tests

**Comment by fcard** [comment]
Yeah, haven't figured out how to make that test yet. I thought about redirecting `STDERR` to test/hide the warnings but the tests being run in parallel complicates that.

Anyway, should I just revert the `apropos` fix in #11438 and post it as a separate PR?

**Comment by tkelman** [comment]
Thought this looked familiar. Maybe split out the `apropos` fix and its test into a separate PR if nobody responds to my comment in https://github.com/JuliaLang/julia/pull/11538#issuecomment-111649678 and merges that. I'm not familiar enough with the doc code to know if there are any downsides to the other changes there.

**Comment by kjenova** [comment]
When searching for "Core" or "Intrinsics", but not "Base" or nonsense capitalized strings like "Foo" or "Blargh", on the latest master I also get the same error. I mentioned this, because "Core" is not prefixed with @ like @assert or @everywhere, which fails in the same way.

**Comment by pao** [comment]
@kjenova, note that when commenting on GitHub issues and using the names of Julia macros, use backticks to quote the macro names to avoid pinging users who happen to use those as their user names. Thanks!

**Comment by tkelman** [comment]
Should be closed by https://github.com/JuliaLang/julia/pull/11538, leave a comment or reopen if not.
